reason-react-native/src/apis/UIManager.rei

```diff
@@ -0,0 +1,29 @@
+type popupAction;
```

Comment:
If we make this an abstract type, I think we should also expose the corresponding constants (UIManagerModuleConstants.ACTION_ITEM_SELECTED and UIManagerModuleConstants.ACTION_DISMISSED on the Java side). Not sure if/where they are exported to JS though.

Reply:
They were exposed as `PopupMenu= { dismissed: 'dismissed', itemSelected: 'itemSelected' }`.